### PR TITLE
Add source regactions registry

### LINKED ISSUE #115: Добавить новый источник "regactions.registry" и новые поля отчета
Вопрос                         | Ответ
------------------------------ | :---:
Наименование группы источников | `default`

### Добавляемые источники

Ниже перечислены источники, которые необходимо добавить в файл спецификации согласно правилам, описанным в [readme-файле][readme]:

```json
{
    "name": "regactions.registry",
    "description": "Реестр Регистрационных действий",
    "enabled": true
}
```

### Добавляемые филды
Ниже перечислены филды, которые необходимо добавить в файл спецификации согласно правилам, описанным в readme-файле:

```json
{
    "path": "tech_data.engine.standarts.emission.euro",
    "description": "Характеристики ТС: Экологический евро стандарт двигателя ",
    "types": [
        "string"
    ],
    "fillable_by": [
        "regactions.registry"
    ]
}
```
```json
{
    "path": "additional_info.vehicle.segment.euro.code",
    "description": "Характеристики ТС: Сегмент автомобиля",
    "types": [
        "string"
    ],
    "fillable_by": [
        "regactions.registry"
    ]
}
```
```json
{
    "path": "additional_info.vehicle.segment.euro.description",
    "description": "Характеристики ТС: Описание сегмента автомобиля",
    "types": [
        "string"
    ],
    "fillable_by": [
        "regactions.registry"
    ]
}
```
[readme]:https://github.com/avtocod/specs/blob/master/README.md

### PR DESCRIPTION
**Pull request by Zaboday**
| Q             | A
| ------------- | ---
| Bug fix?      | No
| New feature?  | Yes

### Added

- Source `regactions.registry` [#115]
- Field with path `tech_data.engine.standarts.emission.euro` [#115]
- Field with path `additional_info.vehicle.segment.euro.code` [#115]
- Field with path `additional_info.vehicle.segment.euro.description` [#115]
- Field with path `registration_actions.items[].actuality.date` [#115]

### Changed

- Field with path `identifiers.vehicle.vin` also fillable by `regactions.registry` source [#115]
- Field with path `identifiers.vehicle.body` also fillable by `regactions.registry` source [#115]
- Field with path `identifiers.vehicle.chassis` also fillable by `regactions.registry` source [#115]
- Field with path `tech_data.brand.name.original` also fillable by `regactions.registry` source [#115]
- Field with path `tech_data.body.number` also fillable by `regactions.registry` source [#115]
- Field with path `tech_data.year` also fillable by `regactions.registry` source [#115]
- Field with path `tech_data.engine.fuel.type` also fillable by `regactions.registry` source [#115]
- Field with path `tech_data.engine.number` also fillable by `regactions.registry` source [#115]
- Field with path `tech_data.engine.volume` also fillable by `regactions.registry` source [#115]
- Field with path `tech_data.engine.power.hp` also fillable by `regactions.registry` source [#115]
- Field with path `tech_data.weight.netto` also fillable by `regactions.registry` source [#115]
- Field with path `tech_data.weight.max` also fillable by `regactions.registry` source [#115]
- Field with path `tech_data.wheel.position` also fillable by `regactions.registry` source [#115]
- Field with path `additional_info.vehicle.category.code` also fillable by `regactions.registry` source [#115]
- Field with path `additional_info.vehicle.category.description` also fillable by `regactions.registry` source [#115]
- Field with path `additional_info.vehicle.owner.type` also fillable by `regactions.registry` source [#115]
- Field with path `additional_info.vehicle.owner.geo` also fillable by `regactions.registry` source [#115]
- Field with path `ownership.history.items[].date.start` also fillable by `regactions.registry` source [#115]
- Field with path `ownership.history.items[].owner.type` also fillable by `regactions.registry` source [#115]
- Field with path `registration_actions.items[].date.start` also fillable by `regactions.registry` source [#115]
- Field with path `registration_actions.items[].date.end` also fillable by `regactions.registry` source [#115]
- Field with path `registration_actions.items[].owner.type` also fillable by `regactions.registry` source [#115]
- Field with path `registration_actions.items[].owner.org.name` also fillable by `regactions.registry` source [#115]
- Field with path `registration_actions.items[].owner.org.tin` also fillable by `regactions.registry` source [#115]
- Field with path `registration_actions.items[].type` also fillable by `regactions.registry` source [#115]
- Field with path `registration_actions.items[].geo` also fillable by `regactions.registry` source [#115]

Closes #115

## Checklist

- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my own code
- [x] I have commented my code, particularly in hard-to-understand areas
- [] I wrote unit tests for my code _(if tests is required for my changes)_
- [x] I have made changes in [CHANGELOG.md](https://github.com/avtocod/specs/blob/master/CHANGELOG.md) file

### CODE REVIEW

File: fields/default/fields_list.json [hunk 1]
```diff
@@ -329,6 +329,16 @@
...
         ]
     },
+    {
+        "path": "tech_data.engine.standarts.emission.euro",
+        "description": "Характеристики ТС: Экологический евростандарт двигателя",
+        "types": [
+            "string"
+        ],
+        "fillable_by": [
+            "regactions.registry"
```
Comment (Reallife) [inline fields/default/fields_list.json]: Need to update `fillable_by` for other fields that may be filled by new source.

Comment (Zaboday) [reply fields/default/fields_list.json]: Fixed for all fields filled by `regactions.registry`.

File: CHANGELOG.md [hunk 2]
```diff
@@ -4,6 +4,16 @@ All notable changes to this project will be documented in this file.
...
+## UNRELEASED
+
+### Added
+
+- Source `regactions.registry` [#115]
+- Field with path `tech_data.engine.standarts.emission.euro` [#115]
+- Field with path `additional_info.vehicle.segment.euro.code` [#115]
+- Field with path `additional_info.vehicle.segment.euro.description` [#115]
+- Field with path `registration_actions.items[].actuality.date` [#115]
+
```
Comment (Reallife) [inline CHANGELOG.md]: Add mentions that some fields also fillable by new source, e.g. 
```
- Field with path `tech_data.engine.standarts.emission.euro` also fillable by `regactions.registry` source [#115]
```

Comment (Zaboday) [reply CHANGELOG.md]: Mentions are added.

**Pull request merged**